chain/src/txhashset/txhashset.rs

```diff
@@ -1323,28 +1323,39 @@ impl<'a> Extension<'a> {
 
 		let mut kern_count = 0;
 		let total_kernels = pmmr::n_leaves(self.kernel_pmmr.unpruned_size());
+		let mut tx_kernels: Vec<TxKernel> = Vec::with_capacity(1_024);
```

Comment:
I think we can simplify this by just using an iterator and `chunks`.
https://doc.rust-lang.org/std/primitive.slice.html#method.chunks

That way we don't need to hand code the chunking logic.

Reply:
sorry for late response, let me finish this `chunks` today.

Reply:
Oops,  the `chunks` is useful for splitting an existing big vector, but here I need create and fill a new vector, and I don't want to create a huge vector and later to split it with `chunks`. So I think it's better to keep it like this.

Pls let me know if thought wrongly.